pkcs11-rev.opam

```diff
@@ -1,4 +1,5 @@
 opam-version: "2.0"
+version: "1.0.1"
```

Comment:
Shouldn’t this be 1.0.2?

Reply:
Nope, this one doesn't work on macos. I didn't want its version to change since nothing changed at all.

Do you think this is not the right thing to do?

Reply:
Why add a `version` indicator here? I don't think we've ever needed to use it.

Reply:
I explained it elsewhere: I didn't want this one updated to 1.0.2 with nothing changed.

But opam guys decided not to change the version in the opam repository so I think we should do the same?